src/BasicDisplays/ModelListDisplay.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from 'react';
+
+import type { FormDisplayChildProps } from '@/Form';
+import { cn } from '@/lib/utils';
+
+export interface ModelListDisplayProps extends FormDisplayChildProps {
+  className?: string;
+  getDisplayValue?: (value: FormDisplayChildProps['value']) => ReactNode;
+}
+
+export const ModelListDisplay = ({
+  value,
+  getDisplayValue = (val) => {
+    return val?._display_value ?? 'None';
```

Comment:
use camelCasing for displayValue

Reply:
Do you mean instead of `_display_value`? or instead of `getDisplayValue`?

Reply:
Yea, change all instances